sdk/core/core-tracing/package.json

```diff
@@ -37,9 +37,9 @@
   },
   "files": [
     "dist/",
-    "dist-esm/lib/",
+    "dist-esm/src/",
+    "src/",
     "src/",
-    "lib/",
     "types/core-tracing.d.ts",
     "ThirdPartyNotices.txt"
```

Comment:
I think we can remove the TPN now since we don't embed OpenTelemetry anymore. Can you remove that file and strike it from the manifest here?

Reply:
done